### PR TITLE
Makes explicit paths in the code/datums/periodic_news.dm file

### PR DESCRIPTION
**Pull request by trololiver112**
## What Does This PR Do

Adds more explicit paths in the `code/datums/periodic_news.dm` file making it cleaner.

## Why It's Good For The Game

Cleaner code and also makes it easier to work with as well as find.

### CODE REVIEW

**Comment by SteelSlayer** [comment]
Oh my god that old format is the most disgusting thing I've ever seen.

**Comment by Fox-McCloud** [comment]
This doesn't need a changelog.

We shouldn't really be having changelogs for things that don't impact players; the changelog is somethign that's primarily meant for player impacting things so they know what has/hasn't change in their game.

**Comment by trololiver112** [comment]
Okay, i removed the changelog

File: code/datums/periodic_news.dm [hunk 1]
```diff
@@ -1,135 +1,123 @@
 // This system defines news that will be displayed in the course of a round.
 // Uses BYOND's type system to put everything into a nice format
 
-// THIS ISNT PROPERLY PATHED AND I AM GOING TO FUCKING SCREAM AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```
Comment (farie82) [inline code/datums/periodic_news.dm:4]: AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA

File: code/datums/periodic_news.dm [hunk 2]
```diff
@@ -1,135 +1,123 @@
...
-			We have little to trade with them now\". Nanotrasen officials have denied these allegations, calling them
-			\"further proof\" of the colony's anti-Nanotrasen stance. Meanwhile, Refuge Security has been unable to quell
-			the riots. More on this at 6."}
-			round_time = 60 * 60
 
+	var/round_time // time of the round at which this should be announced, in seconds
+	var/message // body of the message
+	var/author = "Nanotrasen Editor"
+	var/channel_name = "Nyx Daily"
+	var/can_be_redacted = 0
```
Comment (farie82) [inline code/datums/periodic_news.dm:10]: FALSE

File: code/datums/periodic_news.dm (hunk 1 above)
Comment (trololiver112) [reply code/datums/periodic_news.dm:4]: AAAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by AffectedArc07** [comment]
Remove the :cl: blocks completely, because the generator may break here

**Pull request merged**